interface/src/app/signature/participation-type-selector.tpl.html

```diff
@@ -0,0 +1,7 @@
+<div class="alert alert-danger" role="alert" ng-if="$ctrl.errorMessage">
+  {{$ctrl.errorMessage}}
+</div>
+<select name="selectedParticipationType"
```

Comment:
Ok, done! If you have time to check it out, this is what it looks like now: http://165.123.67.154:8000/#/signature/400